libraries/AP_Airspeed/AP_Airspeed_MS4525.cpp

```diff
@@ -47,6 +47,7 @@ bool AP_Airspeed_MS4525::init()
         uint8_t bus;
         uint8_t addr;
     } addresses[] = {
+        { 2, MS4525D0_I2C_ADDR },
```

Comment:
I think that putting at the end of the list might be marginally better, most boards will find it in bus 0 or 1.